blog/2023-01-09-Apache-Pulsar-2-10-3.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "What’s New in Apache Pulsar 2.10.3"
+date: 2023-01-09
+author: "liangyepianzhou, momo-jun"
+---
+
+The Apache Pulsar community releases version 2.10.3! 50 contributors provided improvements and bug fixes that delivered 155 commits. Thanks for all your contributions.
+
+This blog walks through the most noteworthy changes. For the complete list including all feature enhancements and bug fixes, check out the [Pulsar 2.10.3 Release Notes](https://pulsar.apache.org/release-notes/versioned/pulsar-2.10.3/).
```

Comment:
Please add a `<!-- truncate -->` marker to avoid showing the whole blog on the overview page.

See https://docusaurus.io/docs/blog#blog-list.

Reply:
Thanks for letting me know. Resolved.